Doc updates for My Access - new PR incorporating all feedback from previous reviews (#11361)
microsoftgraph/microsoft-graph-docs: api-reference/beta/api/accesspackage-filterbycurrentuser.md

---

> title: "accessPackage: FilterByCurrentUser"

Reviewer: Please verify the method name - generally, in Graph we use lower camel case, so this should be filterByCurrentUser (lowercase f).

Author: done

description: "Retrieve a list of accesspackage objects filtered on the signed-in user."
localization_priority: Normal
author: "sbounouh"
ms.prod: "microsoft-identity-platform"
doc_type: "apiPageType"
---

# accessPackage: FilterByCurrentUser
Namespace: microsoft.graph

> 

Reviewer: Global: All beta topics need to include the beta disclaimer includes file link; this should be placed on a new line following the namespace line. Please add that to all new beta files.

|Application|Not supported.|

## HTTP request

<!-- {
  "blockType": "ignored"
}
-->
``` http
GET /identityGovernance/entitlementManagement/accessPackages/FilterByCurrentUser
```

## Query parameters
The following table shows the parameters that can be used with this method.

|Parameter|Type|Description|
|:---|:---|:---|
|on|String|The list of options that can be used to filter on current user. Possible option is: `allowedRequestor`. Additional options may be available in the future.|

- `allowedRequestor` is used to get the `accessPackage` objects for which the signed-in user is allowed to submit access requests. The resulting list includes all access packages that can be requested by the caller across all catalogs.

## Request headers
|Name|Description|
|:---|:---|
|Authorization|Bearer {token}. Required.|

## Request body
Do not supply a request body for this method.

## Response

If successful, this method returns a `200 OK` response code and an [accessPackage](../resources/accesspackage.md) collection in the response body.

## Examples

The following example gets the access packages that can be requested by the signed-in user.

> #### Request

Reviewer: FYI - I fixed this while making edits to the topic. When we have one example in the Examples section, we don't add an H3 with a title for the example, the Request and Response subheadings can be H3s. We add H3s with unique titles when we include multiple examples that show different scenarios.

---
title: "accessPackage: filterByCurrentUser"
description: "Retrieve a list of accesspackage objects filtered on the signed-in user."
localization_priority: Normal
author: "sbounouh"
ms.prod: "microsoft-identity-platform"
doc_type: "apiPageType"
---

# accessPackage: filterByCurrentUser
Namespace: microsoft.graph

[!INCLUDE [beta-disclaimer](../../includes/beta-disclaimer.md)]

In [Azure AD Entitlement Management](../resources/entitlementmanagement-root.md), retrieve a list of [accessPackage](../resources/accesspackage.md) objects filtered on the signed-in user.

## Permissions
One of the following permissions is required to call this API. To learn more, including how to choose permissions, see [Permissions](/graph/permissions-reference).

|Permission type|Permissions (from least to most privileged)|
|:---|:---|
|Delegated (work or school account)|EntitlementManagement.Read.All, EntitlementManagement.ReadWrite.All|
|Delegated (personal Microsoft account)|Not supported.|
|Application|Not supported.|

## HTTP request

<!-- {
  "blockType": "ignored"
}
-->
``` http
GET /identityGovernance/entitlementManagement/accessPackages/filterByCurrentUser
```

> ## Query parameters

Reviewer 2: Function parameters

> The following table shows the parameters that can be used with this method.

Reviewer 2: ```suggestion
The following table shows the parameters that can be used with the function.
```

|Parameter|Type|Description|
|:---|:---|:---|

> |on|String|The list of options that can be used to filter on current user. Possible option is: `allowedRequestor`. filterByCurrentUser|

Reviewer 2: ```suggestion
|on|String|The values that can be used to filter on current user. Possible values: `allowedRequestor`. filterByCurrentUser|
```

- `allowedRequestor` is used to get the `accessPackage` objects for which the signed-in user is allowed to submit access requests. The resulting list includes all access packages that can be requested by the caller across all catalogs.

## Request headers
|Name|Description|
|:---|:---|
|Authorization|Bearer {token}. Required.|

## Request body
Do not supply a request body for this method.

## Response

If successful, this method returns a `200 OK` response code and an [accessPackage](../resources/accesspackage.md) collection in the response body.

## Examples
The following example gets the access packages that can be requested by the signed-in user.

### Request
<!-- {
  "blockType": "request",
  "name": "accesspackageassignmentrequest_filterByCurrentUser"
}
-->
``` http
GET https://graph.microsoft.com/beta/identityGovernance/entitlementManagement/accessPackages/filterByCurrentUser(on='allowedRequestor')
```

### Response

> **Note:** The response object shown here might be shortened for readability.

Reviewer: Global: We had a bug in the tool and it didn't generate this note text correctly; we need to add the ">" so that it's formatted as a note.

Reviewer 3: Added ">" to those access package API files which were missing it.
